src/braket/aws/aws_device.py

```diff
@@ -47,6 +47,50 @@ class AwsDeviceType(str, Enum):
     QPU = "QPU"
 
 
+class BraketDevices:
```

Comment:
Considering putting this inside `AwsDevice` as `Devices` so users can just have a single import and write something like `AwsDevice(AwsDevice.Devices.Amazon.SV1`. Thoughts?

Reply:
I think there is going to be some confusion regarding naming. When I see `AwsDevice(BraketDevices.OQC.Lucy)` is it an AWS device? A Braket Device? An  OQC device?

I think the enum should just be `Devices` to help make that clearer: `Devices.<Provider>.<Name>`.

Reply:
Sure, down for `BraketDevices`->`Devices`.

My alternate suggestion would just be moving the enum to the `AwsDevice` class, but still wouldn't require instantiating the object to access. So same as currently proposed, `AwsDevice.Devices.Amazon.SV1` would just give the string ARN for SV1, pulled from the enum and without any service calls.

I don't see the need to break the `AwsDevice(arn)` pattern of instantiating devices here

Reply:
Another benefit of putting it in the `AwsDevice` class is that it resolves some of the scope ambiguity. Technically the local simulator is a Braket "Device", so `AwsDevice.Devices.<etc>` may be a clearer indication of which devices are considered.

As a counterpoint, maybe we should leave it outside of AwsDevice and add `local:braket/default` as the ARN for the local simulator (as prescribed by the jobs interface), but I'm less convinced about the utility of that

Reply:
+1 to Mikes point - `Devices.<Provider>.<Name>` 

Devices without the AWS or Braket as the enum allows us additional flexibility as we scale up in the future - slack me and I can elaborate.

Reply:
Sounds good to me!

Reply:
What about `DeviceIdentifiers`? I like that `Devices` is short and sweet, but the class just contains the ARNs with no additional functionality.

Reply:
I'd probably vote for `Devices` since the "identifiers" part is now somewhat abstracted from the customers. It's true that there's no functionality outside of the ARNs though, but maybe later we can add a class method to a common base class like `.create()` or something, which could have the same interface as the `AwsDevice` constructor